deepreg/predict.py

```diff
@@ -345,6 +364,15 @@ def main(args=None):
         type=str,
     )
 
+    parser.add_argument(
+        "--config_path",
+        "-c",
+        help="Path of config, must endswith .yaml. Can pass multiple paths.",
```

Comment:
Typo "must endswith" -> "must end with"

Reply:
sure, it was bcz the func `endswith` lol